commit bfa4a7b8e689a13492320cee75e7d7f1717b10c8
Author: Szymichowski, Pawel <pawel.szymichowski@intel.com>
Date:   Wed Sep 16 02:49:05 2020 -0700

    Removed clang block type arguments from precompiled builtins in BiFModule.
    
    Change-Id: I3fddae0293830856d3db731fdccd7a631e323d71

diff --git a/IGC/BiFModule/Headers/spirv.h b/IGC/BiFModule/Headers/spirv.h
index 5b8392d2..feb2a29e 100644
--- a/IGC/BiFModule/Headers/spirv.h
+++ b/IGC/BiFModule/Headers/spirv.h
@@ -4573,7 +4573,7 @@ typedef struct
 
 // OpEnqueueKernel will be custom lowered?
 #if (__OPENCL_C_VERSION__ >= CL_VERSION_2_0)
-uint __builtin_spirv_OpGetKernelWorkGroupSize_fp0i32_p0i8_i32_i32(void (^Invoke)(int), private uchar *Param, uint ParamSize, uint ParamAlign);
+uint __builtin_spirv_OpGetKernelWorkGroupSize_fp0i32_p0i8_i32_i32(uchar* Invoke, private uchar *Param, uint ParamSize, uint ParamAlign);
 #endif // __OPENCL_C_VERSION__ >= CL_VERSION_2_0
 
 #if (__OPENCL_C_VERSION__ >= CL_VERSION_2_0)
diff --git a/IGC/BiFModule/Implementation/device_side_enqueue.cl b/IGC/BiFModule/Implementation/device_side_enqueue.cl
index ee3c1aad..52326d02 100644
--- a/IGC/BiFModule/Implementation/device_side_enqueue.cl
+++ b/IGC/BiFModule/Implementation/device_side_enqueue.cl
@@ -47,7 +47,7 @@ uint __builtin_spirv_OpEnqueueMarker_i64_i32_p4i64_p4i64(Queue_t Queue, uint Num
 }
 
 #define DEFN_GET_KERNEL_WORK_GROUP_SIZE(ADDRSPACE_NUMBER, ADDRSPACE_NAME)                                                                                                \
-uint __builtin_spirv_OpGetKernelWorkGroupSize_p0func_p##ADDRSPACE_NUMBER##i8_i32_i32(void (^Invoke)(int), ADDRSPACE_NAME uchar *Param, uint ParamSize, uint ParamAlign)  \
+uint __builtin_spirv_OpGetKernelWorkGroupSize_p0func_p##ADDRSPACE_NUMBER##i8_i32_i32(uchar* Invoke, ADDRSPACE_NAME uchar *Param, uint ParamSize, uint ParamAlign)  \
 {                                                                                                                                                                        \
   return __builtin_IB_get_max_workgroup_size();                                                                                                                          \
 }
